### ISSUE TITLE
Move annotations to sphinx

### ISSUE THREAD

**Issue by thespacedoctor** [thespacedoctor/lasair-web-refresh-issues#19]
An annotator adds extra information, called annotations to Lasair objects.

### Annotators

*   **alerce\_lc**  
    The [ALeRCE light curve classifier](https://arxiv.org/abs/2008.03311) uses variability features computed from the ZTF alert stream, and colors obtained from AllWISE and ZTF photometry. We apply a Balanced Random Forest algorithm with a two-level scheme, where the top level classifies each source as periodic, stochastic, or transient, and the bottom level further resolves each of these hierarchical classes, amongst 15 total classes. This classifier corresponds to the first attempt to classify multiple classes of stochastic variables (including core- and host-dominated active galactic nuclei, blazars, young stellar objects, and cataclysmic variables) in addition to different classes of periodic and transient sources, using real data.   
    Reproduced with permission from the [Alerce Community Broker.](http://alerce.science/).  
    _responsible user: Roy Williams_
*   **alerce\_stamp**  
    The [Alerce Stamp Classifier](https://arxiv.org/abs/2008.03309) is based on a convolutional neural network, trained on alerts ingested from the Zwicky Transient Facility (ZTF). Using only the science, reference and difference images of the first detection as inputs, along with the metadata of the alert as features, the classifier is able to correctly classify alerts from active galactic nuclei, supernovae (SNe), variable stars, asteroids and bogus classes, with high accuracy (94 percent) in a balanced test set.   
    Reproduced with permission from the [Alerce Community Broker.](http://alerce.science/)  
    _responsible user: Roy Williams_
*   **fastfinder**  
    Fastfinder is an early-time, fast transient alerting system in development at Queen's University Belfast. The purpose of Fastfinder is to identify any fast-evolving extragalactic transients, such as Kilonovae, from the LSST alert-stream using only the transient's early-time photometry. By comparing the transient's photometric characteristics to that of the parameter space of known fast-evolving transient classes, Fastfinder generates probabilistic scores on the likelihood of the transient's spectral type.  
    _responsible user: Michael Fulton_
*   **fink\_early\_sn**  
    [Fink](https://fink-broker.org/) is a LSST Community Alert Broker being developed by an international community of researchers with a large variety of scientific interests, including among others multi-messenger astronomy, supernovae, solar system, anomalies identification, microlensing and gamma-ray bursts optical counterparts. This sub-stream focuses on early supernova type Ia. In order to tag an alert as an early supernova Ia candidate, Fink mainly uses 2 criteria: (a) Ia probability larger than 50% from the [early supernova Ia module](https://arxiv.org/abs/2111.11438) , (b) Ia probability larger than 50% from either one of the deep learning classifier based on [SuperNNova](https://arxiv.org/abs/1901.06384). These candidates are also sent nightly to TNS for spectroscopic follow-up. Users should expect about 90% purity based on current performance. See also the [filter code](https://github.com/astrolabsoftware/fink-filters/blob/master/fink_filters/filter_early_sn_candidates/filter.py).   
    Reproduced with permission from the [Fink Community Broker.](https://fink-broker.org/)  
    _responsible user: Roy Williams_
*   **test**  
    Testing  
    _responsible user: Roy Williams_
